Intro for GUC overview (#4301)
timescale/docs: api/configuration/gucs.md

You use the following Grand Unified Configuration (GUC) parameters to optimize the behavior of your $SERVICE_LONG.

The namespace of each GUC is `timescaledb`. 

> Setting a GUC, requires specifing the GUC name with the namespace, as follows:

Reviewer: ```suggestion
To set a GUC you specify `<namespace>.<GUC name>`. For example:
```
